Tensile/KernelWriterAssembly.py

```diff
@@ -10201,7 +10241,7 @@ def __init__(self, kernelWriter, kernel, ss, gwvw, edge, beta, atomic):
 
         if atomic:
           # flat atomics have another VGPR to allow different data for return#
-          regsPerElement = 2 if kernel["BufferStore"] else (3 + 1) # + 1 for alignment
```

Comment:
I forgot to mention regarding this change.
I unified register usage for BufferStore=0 and 1.
No need to allocate a separate set of register for dst of global_atomic_cmpswap.
We can specify same register as src0 for dst.
All tests passed with this change.